.github/issue_labels/unknown_labels.txt

```diff
@@ -0,0 +1,127 @@
+1 min review
```

Comment:
🤔 what is the purpose of this file? I don't think we want to update it every time new label is introduced